kge/job/eval.py

```diff
@@ -132,8 +200,65 @@ def create_from(
         return super().create_from(checkpoint, new_config, dataset, parent_job)
 
 
-# HISTOGRAM COMPUTATION ###############################################################
+class TrainingLossEvaluationJob(EvaluationJob):
+    """ Entity ranking evaluation protocol """
+
+    def __init__(self, config: Config, dataset: Dataset, parent_job, model):
+        super().__init__(config, dataset, parent_job, model)
+        self.is_prepared = True
```

Comment:
Don't do this. Simply use an empty _prepare method. Perhaps you can even let it emtpy in the supervlass instead of using NotImplementedError